optimum/exporters/neuron/base.py

```diff
@@ -174,6 +175,7 @@ def __init__(
         self._config = config
         self._normalized_config = self.NORMALIZED_CONFIG_CLASS(self._config)
         self.mandatory_axes = ()
+        self.tp_degree = tensor_parallel_size
```

Comment:
Hummm ... see my comment above

Reply:
Now I find it confusing to use a different name. Especially since you change it back to tensor_parallel_size when storing the config

Reply:
I will use tensor_parallel_size as the config property as well.